### ISSUE TITLE
Thoroughly Document Developer Process for Testing Coded Additions to Splashkit

### ISSUE THREAD

**Issue by njsch** [splashkit/splashkit-core#127]
So far this does not appear to have been done in any great detail - I am still learning the ropes myself.  It really should be documented somewhere as to how exactly to create and use test code.  It's not actually as simple as writing a new library for modular inclusion: you have to also add_test to test_main.cpp and reflect this accordingly within the source file you have made changes to.  Because this does not appear to be clarified, you have to manually locate the relevant files and then read the code to figure it out.  This is not difficult but would take up less time with documentation.

**Comment by macite** [comment]
Yes it would be great to get this documented. If anyone has time I'm happy to review and help contribute. Time is the big issue for me at the moment.
